osc/build.py

```diff
@@ -1143,7 +1143,10 @@ def __str__(self):
         if bi.installonly_list:
             rpmlist.append('installonly: ' + ' '.join(bi.installonly_list) + '\n')
 
-    rpmlist_file = NamedTemporaryFile(prefix='rpmlist.')
+    if sys.version_info >= (3, 0):
+      rpmlist_file = NamedTemporaryFile(mode='w+t', prefix='rpmlist.')
```

Comment:
This variant works in both Python 2 and Python 3, so we can simplify to that.

Reply:
yes. you are right.